sys/net/gnrc/network_layer/ipv6/nib/Kconfig

```diff
@@ -0,0 +1,108 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+menuconfig KCONFIG_MODULE_GNRC_IPV6_NIB
+    bool "Configure GNRC IPv6 NIB"
+
+if KCONFIG_MODULE_GNRC_IPV6_NIB
+
+config GNRC_IPV6_NIB_CONF_6LBR
+    bool "6Lo border router features"
+    default y if MODULE_GNRC_IPV6_NIB_6LBR
```

Comment:
The `6LN`, `6LR` and `6LBR` should contain at least references to https://tools.ietf.org/html/rfc6775#section-2. Otherwise, nobody will know or understand what this is about.